### ISSUE TITLE
Tech: Group#worlds should probably be a Set instead of List

### ISSUE THREAD

**Issue by ljacqu** [Gnat008/PerWorldInventory#159]
Found in Group.java:
```java
    public void addWorlds(List<String> worlds) {
        for (String world : worlds)
            if (!this.worlds.contains(world))
                this.worlds.add(world);
    }
```

It would be easier to just keep `Group#worlds` as a `Set<String>` instead of `List<String>` -->
```java
    public void addWorlds(Collection<String> worlds) {
        this.worlds.addAll(worlds);
    }
```

**Comment by MrOpposite** [comment]
I second this! I noticed this during my development of the last-position-thing and thought that the worlds ought to be a set, just as my last-world/group settings are. I just found that fix out of scope for the pull request in question and decided to go back and fix that once the PR is either merged or rejected, as including that in the current PR is out of scope, and creating a new one based on current master will lead to horrible merge conflicts if my PR is accepted.
